src/wix/WixToolset.Core.WindowsInstaller/Unbind/ExtractCabinetsCommand.cs

```diff
@@ -55,13 +55,21 @@ public void Execute()
             // index all of the cabinet files
             if (OutputType.Module == this.Output.Type || this.TreatOutputAsModule)
             {
+                var mediaRow = new MediaRow(null, WindowsInstallerTableDefinitions.Media);
+                mediaRow[0] = 1;
+                mediaRow[1] = 1;
+                mediaRow[3] = "MergeModule.CABinet";
```

Comment:
Could use the named properties here instead of direct column access: `.DiskId`, `.LastSequence`, `.Cabinet`.